Client/Ecosia/Migration/EcosiaImport.swift

```diff
@@ -36,56 +29,34 @@ final class EcosiaImport {
     }
 
     let profile: Profile
-    let tabManager: TabManager
-    private var waitForTabs = false
-    private var tabMigrationStart = Date()
-    private let migrationGroup = DispatchGroup()
 
-    private let migration = Migration()
-    private var finished: ((Migration) -> ())?
     private var progress: ((Double) -> ())?
 
-
-    init(profile: Profile, tabManager: TabManager) {
+    init(profile: Profile) {
         self.profile = profile
-        self.tabManager = tabManager
     }
 
-    private var tabsProgress = 0.0 { didSet { progress?(totalProgress) } }
     private var favsProgress = 0.0 { didSet { progress?(totalProgress) } }
     private var historyProgress = 0.0 { didSet { progress?(totalProgress) } }
 
     private var totalProgress: Double {
-        return (tabsProgress + favsProgress + historyProgress) / 3.0
+        return (favsProgress + historyProgress) / 2.0
     }
 
     func migrate(progress: ((Double) -> ())? = nil, finished: @escaping (Migration) -> ()) {
-        self.finished = finished
         self.progress = progress
 
-        // Tab migration is triggered by Browser setup in parallel, so it can be it has finished before we get here
-        let ecosiaTabs = Core.Tabs().items
```

Comment:
We are not migrating tabs anymore?

Reply:
we do - but it's happening outside of this migration abstraction. This code here was only used to monitor the tab migration but the monitoring does not work as expected, so let's remove it. 

Let me point you where the tab migration is happening.
Here it is triggered:
https://github.com/ecosia/ios-browser/blob/3c1ea1197b2db3acef96e2bb6887e22be1947971/Client/Frontend/Browser/TabManagerStore.swift#L22

There it migrates:
https://github.com/ecosia/ios-browser/blob/3c1ea1197b2db3acef96e2bb6887e22be1947971/Client/Frontend/Browser/TabManagerStore.swift#L162

It "just" hooks into the original Tab restoration mechanics which has not produced any errors for the first batches of migrations and therefore needsno extra monitoring any more, agree?

Reply:
Thanks for clarifying. Looks neat 💯